app/grandchallenge/cases/models.py

```diff
@@ -174,12 +174,12 @@ def update_status(
                 description=self.error_message,
                 action_object=self,
             )
-        from grandchallenge.algorithms.models import Job
+        from grandchallenge.components.models import ComponentJob
 
         if (
             self.error_message
             and linked_object
-            and isinstance(linked_object, Job)
+            and isinstance(linked_object, ComponentJob)
```

Comment:
Are we certain that these messages will not leak any information to the challenge participants?

Reply:
I would probably err on the side of caution, just in case they are private or we change things later so that all inputs somehow go through this route.

Reply:
Ok, are you suggesting to replace the specific error message with a generic "input validation failed" instead? But the challenge admins should probably be able to see the full error message. Would it be an idea to hide the error messages of evaluations from participants altogether?

Reply:
Generic would be fine, likely we're not going to hit this as likely we're just using JSON configuration to these things which are going to be validated on the client side too.

Reply:
That's true. Thinking about whether to log the error to sentry so that we have it on record - is that worth it in your eyes ?We'd need to take care to only do this for evaluations or else we would spam sentry.

Reply:
Probably fine to log it to Sentry given that we do not expect for it to occur!

Reply:
I have updated the error handlers accordingly and also the tests.